src/Workleap.OpenApi.MSBuild/ProcessWrapper.cs

```diff
@@ -0,0 +1,25 @@
+using CliWrap;
+using CliWrap.Buffered;
+
+namespace Workleap.OpenApi.MSBuild;
+
+internal sealed class ProcessWrapper : IProcessWrapper
+{
+    private readonly string _workingDirectory;
+
+    public ProcessWrapper(string workingDirectory)
+    {
+        this._workingDirectory = workingDirectory;
+    }
+
+    public async Task<int> RunProcessAsync(string filename, string[] arguments, CancellationToken cancellationToken)
```

Comment:
Just a thought: right now we "silence" the stdout and stderr of the processes. Could it be help for debugging purposes if we print the output to the build log?

I mean, if something goes wrong and the developer doesn't know what happened, then he will reach out to us and we'll have to spend time debugging with him/her, right?